src/System.Private.ServiceModel/src/Internals/WcfEventSource.cs

```diff
@@ -12,6 +12,38 @@ internal sealed class WcfEventSource : EventSource
     {
         public static WcfEventSource Instance = new WcfEventSource();
 
+        // The following four dummy events are to set the correct values for the eventlog channel keyword values. EventSource allocates
+        // the channel id's in the order that they are first used in the event methods lexically. So the first method attributed with [Event]
+		// which specifies an EventChannel will have the first channel keyword assigned to that Channel. We need EventSource to assign
+		// keyword bit 63 to Admin, bit 62 to Operational, bit 61 to Analytic and bit 60 to Debug.
+        [Event(EventIds.DummyAdminEventId, Level = EventLevel.Critical, Channel = EventChannel.Admin, Opcode = EventOpcode.Info, Task = EventTask.None,
+    Keywords = Keywords.DummyKeyword)]
+        private void AADummyAdminChannelEvent()
+        {
+            WriteEvent(EventIds.DummyAdminEventId);
+        }
+
+        [Event(EventIds.DummyOperationalEventId, Level = EventLevel.Critical, Channel = EventChannel.Operational, Opcode = EventOpcode.Info, Task = EventTask.None,
+    Keywords = Keywords.DummyKeyword)]
+        private void ABDummyOperationalChannelEvent()
+        {
+            WriteEvent(EventIds.DummyOperationalEventId);
+        }
+
+        [Event(EventIds.DummyAnalyticEventId, Level = EventLevel.Critical, Channel = EventChannel.Analytic, Opcode = EventOpcode.Info, Task = EventTask.None,
+    Keywords = Keywords.DummyKeyword)]
+        private void ACDummyAnalyticChannelEvent()
+        {
+            WriteEvent(EventIds.DummyAnalyticEventId);
+        }
+
+        [Event(EventIds.DummyDebugEventId, Level = EventLevel.Critical, Channel = EventChannel.Debug, Opcode = EventOpcode.Info, Task = EventTask.None,
```

Comment:
In addition to leaving out the keywords property  (see comment below), I would also leave out the Opcode and level since the defaults are fine it is is less code.   For what it is worth...